device-provisioner/provisioner.sh

```diff
@@ -204,9 +205,9 @@ case ${RPI_DEVICE_FAMILY} in
         ;;
 esac
 
-cp /usr/share/misc/fastboot-gadget.img "${RPI_SB_WORKDIR}"/boot.img
```

Comment:
Might be worth using systemd's RuntimeDirectory= ( $RUNTIME_DIRECTORY here) mechanism?

Reply:
Also not in favour, as RPI_SB_WORKDIR is an optional component that we advise people to create only after they've iterated their image to a release candidate.